tox.ini

```diff
@@ -0,0 +1,41 @@
+[tox]
+skipsdist=True
+skip_missing_interpreters = True
+envlist = lint, unit, integration
```

Comment:
if you run tox without `-e` it may fail because `integration` does not exist yet. not a big deal.

Reply:
Ops again. Evils of cut and paste. Fixed.